_rules/auto-update-text-efbfc7.md

```diff
@@ -13,32 +13,32 @@ accessibility_requirements: # Remove whatever is not applicable
 input_aspects:
   - DOM Tree
   - CSS Styling
-acknowledgements:
+acknowledgments:
   authors:
     - Carlos Duarte
 ---
 
 ## Applicability
 
-The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
+The rule applies to any [visible][] [HTML element][] if:
 
-- **(not alone)** it is not the only [content][] in the [HTML document][]; and
 - **(changed)** the `innerText` property of the [element][html element] changes; and
 - **(no child changed)** the [element][html element] does not have [children][child] whose `innerText` property also changes; and
+- **(not alone)** the [element][html element] has [sibling][] or [ancestor][] elements with non-empty `innerText` properties; and
 - the [document][html document] the [element][html element] belongs to:
   - **(ready)** has [readiness][document readiness] equal to "complete" before the change happens; and
   - **(inactive)** is associated with a [window][] that has the [last activation timestamp][] equal to "positive infinity".
 
-## Expectation 1
+## Expectation
 
-For the test target there exists a [mechanism][] to achieve one of the following objectives:
+For the test target there exists an [instrument][] in the same [HTML document][] of the test target, or in another [HTML document][] that is [linked][hyperlink] from the [HTML document][] of the test target, to achieve one of the following objectives:
```

Comment:
Do we want to require conditions on the link (visible, accessible, …) or not?

Reply:
There is nothing in SC 2.2.2 about that. In other rules where that was the case, and where I had included similar requirements in the expectation, I had to take them out following reviewers' requests. That's why I left those out in this one.